docs/architecture/architecture.md

```diff
@@ -106,7 +109,39 @@ This transition has been delayed until after the release of Clear Containers 3.0
 due to regressions, as described in [runtime issue 407](https://github.com/clearcontainers/runtime/issues/407).
 Once support for features like hotplug are available in `q35`, the project will
 look to transition to this machine type.
- 
+
+### Assets
+
+Additional resources are required to create each QEMU virtual machine
+instance, specifically a guest kernel and a root filesystem image known as the
+"mini O/S".
+
+The locations of these resources are passed to
+QEMU by the runtime (see [Configuration](configuration)).
```

Comment:
I think you need a # on the front of the section name to get it to render to the section?

Reply:
Fixed.